dist/Interfaces/subscription/SubscriptionAddon.d.ts

```diff
@@ -0,0 +1,19 @@
+import { SubscriptionAddonPricingPlan } from './SubscriptionAddonPricingPlan';
+export interface APISubscriptionAddon {
```

Comment:
This interface isn't really needed as they get transformed anyway. So just `SubscriptionAddon` is fine